src/index.js

```diff
@@ -67,12 +66,29 @@ const createContext = (ctx) => {
   return ctx
 }
 
-const importDefault = async filepath => {
-  const module = await import(url.pathToFileURL(filepath).href)
-  return module.default
+/** @type {import('jiti').JITI | null} */
+let jiti = null
+
+const loader = async filepath => {
+  try {
+    const module = await import(url.pathToFileURL(filepath).href)
+    return module.default
+  } catch {
+    if (!jiti) {
+      try {
+        jiti = (await import('jiti')).default(__filename, { interopDefault: true })
+      } catch (err) {
+        /* c8 ignore next 4 */
+        throw new Error(
+          `'jiti' is required for the TypeScript configuration files. Make sure it is installed\nError: ${err.message}`
```

Comment:
We will show this error even if there was no TS files in the system. Can we use `filepath` to check that file is `.tx*`?

Reply:
Yeah makes sense. I'll update that.

Reply:
Got it.